pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -515,14 +528,16 @@ def extract_basic_trig_properties(trial_dict, trigs, slide_dict, seg_dict):
     trig_snr = {}
     trig_bestnr = {}
     for slide_id in slide_dict:
-        slide_trigs = sorted_trigs[slide_id]
-        if slide_trigs:
-            trig_time[slide_id] = trigs['network/end_time_gc'][slide_trigs]
-            trig_snr[slide_id] = trigs['network/coherent_snr'][slide_trigs]
-        else:
-            trig_time[slide_id] = numpy.asarray([])
-            trig_snr[slide_id] = numpy.asarray([])
-        trig_bestnr[slide_id] = trigs['network/reweighted_snr'][slide_trigs]
+        event_indices = [numpy.where(trigs['network/event_id'] == event_id)[0][0] 
+                         for event_id in sorted_trigs[slide_id]]
+        trig_time[slide_id] = numpy.array([trigs['network/end_time_gc'][event_index]
+                               for event_index in event_indices])
+        trig_snr[slide_id] = numpy.array([trigs['network/coherent_snr'][event_index]
+                              for event_index in event_indices])
+        trig_bestnr[slide_id] = numpy.array([1 if trigs['/network/reweighted_snr'][event_index] 
+                                             >= opts.newsnr_threshold
+                                             else 0 for event_index in event_indices])
+        trig_bestnr[slide_id][trig_bestnr[slide_id] < opts.newsnr_threshold] = 0
```

Comment:
This affects `pycbc_pygrb_plot_stats_distribution` and `pycbc_pygrb_efficiency`.  We need to check these changes and how the propagate, very carefully.  In any case, let's first deal with the review requests above and push PR #4679.

Reply:
Please revert to the old code, since it's been tested.